Vision/AutoMLVisionClassifier/app.py

```diff
@@ -0,0 +1,76 @@
+import os
+from typing import Dict, List
+
+import uvicorn
+from dotenv import load_dotenv
+from fastapi import FastAPI, Security, Depends, HTTPException
+from fastapi.security.api_key import APIKeyHeader, APIKey
+from pydantic import BaseModel
+from starlette.status import HTTP_403_FORBIDDEN
+
+from powerskill import extractor, models
+import logging
+
+load_dotenv()
+app = FastAPI()
+
+
+class Values(BaseModel):
+    values: List = []
+
+
+class Value(Values):
+    recordId: str
+    data: Dict[str, str] = None
+
+
+API_KEY = os.environ['KEY']
+API_KEY_NAME = "Ocp-Apim-Subscription-Key"
+
+api_key_header = APIKeyHeader(name=API_KEY_NAME, auto_error=False)
+class_model = models.Models(azureml_model_dir=None, classication_model=None)
+
+
+@app.on_event("startup")
+async def startup_event():
+    try:
+        if os.environ['GET_LATEST_MODEL'].lower() == "true":
+            logging.info(f"Download latest model")
+            if class_model.get_latest_model(os.environ['EXPERIMENT_NAME']):
+                class_model.load_classification_model('models/train_artifacts/')
+            else:
+                class_model.load_classification_model(os.environ['AZUREML_MODEL_DIR'])
```

Comment:
Move environment variable names to constants next to the one you already have.

Reply:
Implemented